petastorm/tests/test_tf_utils.py

```diff
@@ -46,13 +48,17 @@ def _tf_session():
         coord.join(threads)
 
 
-def test_empty_dict():
-    # Check two types that should be promoted/converted (uint16 and Decimal) and one that should not be
-    # modified (int32)
+def test_sanitize_field_tf_types():
+    expected_datetime_array = [datetime.date(1970, 1, 1), datetime.date(2015, 9, 29)]
+    expected_datetime_ns_from_epoch = [timegm(dt.timetuple()) * 1000000000 for dt in expected_datetime_array]
```

Comment:
Wanna add a test with non-zero nanoseconds?

Reply:
I don't think datetime gets to the nanoseconds resolution...